packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react'
+import {
+  ComponentSlotStylesInput,
+  ICSSInJSStyle,
+  SiteVariablesPrepared,
+  NaturalColors,
+} from '../../../types'
+import { AlertProps } from '../../../../components/Alert/Alert'
+import { AlertVariables } from './alertVariables'
+
+const getIntentColorsFromProps = (
+  p: AlertProps,
+  v: AlertVariables,
+  siteVars: SiteVariablesPrepared,
+): React.CSSProperties => {
+  const { colors } = siteVars
+  const naturalColors: NaturalColors = siteVars.naturalColors
+
+  if (p.danger) {
+    return {
+      color: colors.red[900], // $app-red
+      backgroundColor: colors.red[50], // $app-red-10
+      borderColor: colors.red[100], // $app-red-08
+    }
+  }
+
+  if (p.info) {
+    return {
+      color: colors.grey[900], // $app-black
+      backgroundColor: colors.grey[300], // $app-gray09
+      borderColor: colors.grey[400], // $app-gray08
+    }
+  }
+
+  if (p.success) {
+    return {
+      color: colors.green[900], // $app-green-04
+      backgroundColor: colors.grey[50], // $app-white
+      borderColor: naturalColors.lightGreen[900], // $app-green
+    }
+  }
+
+  if (p.warning) {
+    return {
+      color: colors.grey[700],
+      backgroundColor: colors.grey[50], // $app-white
+      borderColor: colors.yellow[900], // $app-yellow
```

Comment:
I think that this should this part of variables:
```
Record<'info', 'danger', { color: string, backrgroundColor: string }>
```

Reply:
agreed but our convention is to have a flat structure for variables for now; we had this discussion before in #756 (see: https://github.com/stardust-ui/react/pull/756#discussion_r249983095)

@levithomason looks like the need for nested variables is back

Reply:
So, may be we can just use something like `colorBackrgroundRootDanger`, https://amzn.github.io/style-dictionary/#/properties?id=category-type-item

@mnajdova do you have any agreement on variables?

Reply:
I am starting the work for refactoring the color palette and color scheme. For now I would rather leave it as is, and would revert the changed color palette, so that we will have easier migration from here.

Reply:
@layershifter 
I believe for the users it's easier to edit the `backgroundColor` variable
```jsx
<Alert variables={{ colorBackrgroundRootDanger: 'red' }} ... />
```

rather than adding the `danger` prop and a strange variable name like `colorBackrgroundRootDanger`:
```jsx
<Alert danger variables={{ colorBackrgroundRootDanger: 'red' }} ... />
```

In this way, we also avoid introducing **15 variables** from **5 variations** (default, info, danger, success, warning) X **3 colors** (text, background, border). Instead we introduce just 3 variables (`color`, `backgroundColor`, `borderColor`),

So I'd also vote to keep it like this.

Reply:
Agree, let's skip it for now.